sdk/core/azure-core/src/http/telemetry_policy.cpp

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <http/policy.hpp>
+
+#include <sstream>
+
+#ifdef _WIN32
+
+#define NOMINMAX
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
+
+namespace {
+
+std::string GetOSVersion()
+{
+  std::ostringstream osVersionInfo;
+
+  {
+    HKEY regKey{};
+    auto regKeyOpened = false;
+    try
+    {
+      if (RegOpenKeyExA(
+              HKEY_LOCAL_MACHINE,
+              "SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion",
+              0,
+              KEY_READ,
+              &regKey)
+          == ERROR_SUCCESS)
+      {
+        regKeyOpened = true;
+
+        auto first = true;
+        char const* regValues[]{
```

Comment:
The array of values we look for should be "static constexpr"

Reply:
Done.